core/frontend/src/WebViewerApp.ts

```diff
@@ -41,8 +42,15 @@ export class WebViewerApp {
       BentleyCloudRpcManager.initializeClient(params.rpcParams, opts.iModelApp?.rpcInterfaces ?? [], params.routing);
     }
     await IModelApp.startup(opts.iModelApp);
-    if (opts.webViewerApp.authConfig)
-      IModelApp.authorizationClient = new BrowserAuthorizationClient(opts.webViewerApp.authConfig);
+
+    if (opts.webViewerApp.authConfig) {
+      await BrowserAuthorizationCallbackHandler.handleSigninCallback(opts.webViewerApp.authConfig.redirectUri);
+      const auth = new BrowserAuthorizationClient(opts.webViewerApp.authConfig);
+      IModelApp.authorizationClient = auth;
+      try {
+        await auth.signInSilent(new ClientRequestContext());
+      } catch (err) { }
+    }

```

Comment:
probably should have some option for silent login. Probably should log that it failed.

Reply:
Done.